h/api/storage.py

```diff
@@ -209,12 +160,8 @@ def delete_annotation(request, id_):
     :param id_: the annotation ID
     :type id_: str
     """
-    if _postgres_enabled(request):
-        annotation = fetch_annotation(request, id_, _postgres=True)
-        request.db.delete(annotation)
-
-    legacy_annotation = fetch_annotation(request, id_, _postgres=False)
-    legacy_annotation.delete()
+    annotation = fetch_annotation(request, id_)
+    request.db.delete(annotation)
```

Comment:
No need to change this now, but this should now really be doing something like `request.db.query(Annotation).filter_by(id=id_).delete()` rather than fetching the thing first.